### ISSUE TITLE
Delivery mode: 'Do not package' is not included in build

### ISSUE THREAD

**Issue by kreys** [jelte/be.khepri.play.assetdelivery.addressables#25]
If you choose delivery mode `Do not package`, then this asset group won't be included into build, nor as part of base.apk, nor as part of any other bundle.

**Comment by jelte** [comment]
true, but you don't need to add the AssetPackGroupSchema to that group. 

The value comes from Google's enum, which is why it is in the list. But it is basically useless. 

I will add it to be ignored in the bundle stripping
